.github/workflows/openlogin.login-with-google.yml

```diff
@@ -66,6 +66,22 @@ jobs:
         if: steps.download-state.outcome == 'success'
         run: yarn run playwright test --config openlogin/login-with-google/index.config.ts --project ${{ matrix.browser }}
 
+      - id: upload-state
+        name: Upload logged-in state once a day
+        run: |
+          last_updated=$(aws s3 ls s3://torus-e2e-tests/openlogin/login-with-google/${{ matrix.browser }}.json | awk '{print $1}')
```

Comment:
So you are retrieving existing snapshot from S3 and checking its updated at time to determine whether or not to refresh.
Since this test suite mandates a pre-existing snapshot to exist on S3 (since it contains the device share) we don't have to handle the case where no snapshot exists (since it is a hard requirement for this test suite to pass).

Reply:
Yeah, you're correct